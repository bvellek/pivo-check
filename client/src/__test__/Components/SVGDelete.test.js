import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import SVGDelete from '../../js/components/SVGDelete';

describe('<SVGDelete />', () => {
  it('renders without crashing', () => {
    shallow(<SVGDelete />);
  });
  it('it renders correctly', () => {
    const component = shallow(
      <SVGDelete />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
