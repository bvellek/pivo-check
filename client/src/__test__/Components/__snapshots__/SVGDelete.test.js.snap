exports[`<SVGDelete /> it renders correctly 1`] = `
<svg
  className="svg-delete"
  viewBox="0 0 24 25"
  xmlns="http://www.w3.org/2000/svg">
  <path
    d="M16.377,12.519 L22.944,5.953 C23.906,4.99 23.906,3.414 22.944,2.451 L22.068,1.576 C21.105,0.612 19.529,0.612 18.567,1.576 L12,8.142 L5.433,1.575 C4.471,0.612 2.895,0.612 1.932,1.575 L1.056,2.45 C0.094,3.413 0.094,4.989 1.056,5.952 L7.622,12.518 L1.056,19.085 C0.094,20.048 0.094,21.623 1.056,22.586 L1.932,23.462 C2.895,24.425 4.471,24.425 5.433,23.462 L12,16.896 L18.567,23.462 C19.529,24.425 21.105,24.425 22.068,23.462 L22.944,22.586 C23.906,21.623 23.906,20.048 22.944,19.085 L16.377,12.519 Z"
    fill="#C54033" />
</svg>
`;
